bin/build-combined-ckan-mo.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
-
-HERE=`dirname $0`
-msgcat --use-first \
-    "$HERE/../i18n/de/LC_MESSAGES/ckanext-switzerland.po" \
-    "$HERE/../../../ckan/ckan/i18n/de/LC_MESSAGES/ckan.po" \
-    | msgfmt - -o "$HERE/../../../ckan/ckan/i18n/de/LC_MESSAGES/ckan.mo"
-msgcat --use-first \
-    "$HERE/../i18n/fr/LC_MESSAGES/ckanext-switzerland.po" \
-    "$HERE/../../../ckan/ckan/i18n/fr/LC_MESSAGES/ckan.po" \
-    | msgfmt - -o "$HERE/../../../ckan/ckan/i18n/fr/LC_MESSAGES/ckan.mo"
-msgcat --use-first \
-    "$HERE/../i18n/it/LC_MESSAGES/ckanext-switzerland.po" \
-    "$HERE/../../../ckan/ckan/i18n/it/LC_MESSAGES/ckan.po" \
-    | msgfmt - -o "$HERE/../../../ckan/ckan/i18n/it/LC_MESSAGES/ckan.mo"
```

Comment:
Remember that this script is currently used in the deployment, so it must be removed there as well.

Reply:
This is taken care of here: https://gitlab.liip.ch/jazz/ogd-ch/blob/0e13bf591a0123f4f189c948e44f88418ed8fcb6/fabfile.py#L231-233

Found no other occurence of this script.